<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>450</height> <!-- Aumenté la altura para acomodar los LEDs -->
   </rect>
  </property>
  <property name="windowTitle">
   <string>Esclavo Modbus TCP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPressure">
      <property name="text">
       <string>Presión:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayoutPressure">
      <item row="0" column="0">
       <widget class="QSlider" name="sliderPressure">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="layoutPressureLimits">
        <item>
         <widget class="QLabel" name="labelPressureMin">
          <property name="text">
           <string>950</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerPressure">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelPressureMax">
          <property name="text">
           <string>1050</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="layoutPressureValue">
        <item>
         <widget class="QLineEdit" name="editPressure"/>
        </item>
        <item>
         <widget class="QLabel" name="labelPressureUnit">
          <property name="text">
           <string>hPa</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTemperature">
      <property name="text">
       <string>Temperatura:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayoutTemperature">
      <item row="0" column="0">
       <widget class="QSlider" name="sliderTemperature">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="layoutTemperatureLimits">
        <item>
         <widget class="QLabel" name="labelTemperatureMin">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerTemperature">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTemperatureMax">
          <property name="text">
           <string>200</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="layoutTemperatureValue">
        <item>
         <widget class="QLineEdit" name="editTemperature"/>
        </item>
        <item>
         <widget class="QLabel" name="labelTemperatureUnit">
          <property name="text">
           <string>°C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="buttonDiscrete">
        <property name="text">
         <string>Botón 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDiscrete2">
        <property name="text">
         <string>Botón 1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayoutLEDs">
      <item>
       <widget class="QFrame" name="ledRed">
       <property name="minimumSize">
        <size>
         <width>50</width> 
         <height>50</height>
        </size>
       </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ledBlue">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ledGreen">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: black;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTreeWidget" name="treeWidgetModbus">
      <property name="headerLabels" stdset="0">
       <stringlist>
        <string>Índice</string>
        <string>Valor</string>
       </stringlist>
      </property>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
